libs/@hashintel/petrinaut/src/petrinaut-sdcpn/views/Editor/components/LeftSideBar/types-section.tsx

```diff
@@ -5,6 +5,50 @@ import { FaChevronDown, FaChevronRight } from "react-icons/fa6";
 import { useEditorStore } from "../../../../state/editor-provider";
 import { useSDCPNStore } from "../../../../state/sdcpn-provider";
 
+// Pool of 10 well-differentiated colors for types
+const TYPE_COLOR_POOL = [
+  "#3b82f6", // Blue
+  "#ef4444", // Red
+  "#10b981", // Green
+  "#f59e0b", // Amber
+  "#8b5cf6", // Violet
+  "#ec4899", // Pink
+  "#14b8a6", // Teal
+  "#f97316", // Orange
+  "#6366f1", // Indigo
+  "#84cc16", // Lime
```

Comment:
@lunelson We have a color palette in the design system ([internal link](https://www.figma.com/design/WmnosvOvi4Blw0HK0jh1mG/Design-System?node-id=37810-47669&t=MoGIlNNsS3s2FoF5-4)) which we could use here (specifically for use in data viz situations like this).

@kube Do you know if these are represented in code anywhere already yet?

Fine to use these as a placeholder/stop-gap, but we want to avoid having to define these across all data viz components we build ideally.